### PR TITLE
[Glimmer2] Add tests for bound {{loc}} behavior

### LINKED ISSUE #13132: Add tests for bound {{loc}} behavior
`{{log boundString boundArg}}` is supported in Ember, however it is not tested. Introduce tests for this per the INUR description here: https://github.com/emberjs/ember.js/issues/13127

See: https://github.com/emberjs/ember.js/pull/13129#issuecomment-198777298

### PR DESCRIPTION
**Pull request by code0100fun**
Tests INUR cycle for bound params. Fixes #13132.

### CODE REVIEW

File: packages/ember-glimmer/tests/integration/helpers/loc-test.js [hunk 1]
```diff
@@ -36,4 +38,47 @@ moduleFor('Helpers test: {{loc}}', class extends RenderingTest {
     this.assertText('Hello, Mr. Pitkin', 'the formatted string is correct after rerender');
   }
 
+  ['@test it updates when bound params change']() {
+    let context = { simple: 'Hello Friend', personal: 'Hello' };
+
+    this.render(`{{loc simple}} - {{loc personal 'Mr. Pitkin'}}`, context);
```
Comment (chancancode) [inline packages/ember-glimmer/tests/integration/helpers/loc-test.js]: you can just do `this.render(`{{...}}`, { simple: ... });`

File: packages/ember-glimmer/tests/integration/helpers/loc-test.js [hunk 2]
```diff
@@ -36,4 +38,47 @@ moduleFor('Helpers test: {{loc}}', class extends RenderingTest {
...
+
+    this.runTask(() => set(this.context, 'simple', 'Hello Friend'));
+    this.assertText('Hallo Freund - Hallo, Mr. Pitkin',
+                    'the bound value is correct after reset');
+  }
+
+  ['@test it updates when nested bound params change']() {
+    let context = { greetings: { simple: 'Hello Friend', personal: 'Hello' } };
+
+    this.render(`{{loc greetings.simple}} - {{loc greetings.personal 'Mr. Pitkin'}}`, context);
```
Comment (chancancode) [inline packages/ember-glimmer/tests/integration/helpers/loc-test.js]: ditto

**Comment by mixonic** [comment]
Looks good :muscle: Thanks @code0100fun!

**Pull request merged**